testing/services/dummy/api-gateway/go.mod

```diff
@@ -6,18 +6,13 @@ replace github.com/ukama/ukama/systems/common => ../../../../systems/common
 
 replace github.com/ukama/ukama/testing/services/dummy-node/api-gateway => ./
```

Comment:
The package should be updated from `github.com/ukama/ukama/testing/services/dummy-node/api-gateway` to `github.com/ukama/ukama/testing/services/dummy/api-gateway`.